extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -108,6 +127,75 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
   return openAttachmentEl;
 };
 
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+ const renderOutlinkPageAttachmentUI = (win, pageEl, attachmentEl, attachmentHref) => {
+  const openAttachmentEl = buildOpenOutlinkAttachmentElement(pageEl);
+
+  // Copy href to the element so it can be previewed on hover and long press.
+  if (attachmentHref) {
+    openAttachmentEl.setAttribute('href', attachmentHref);
+  }
+  
+  openAttachmentEl.addEventListener('click', () =>
+    openAttachmentEl.openAttachment()
+  );
+
+  // Setting theme  
+  const theme = attachmentEl.getAttribute('theme');
+  if (theme && AttachmentTheme.DARK === theme.toLowerCase()) {
+    openAttachmentEl.setAttribute('theme', AttachmentTheme.DARK);
+  }
+
+  // Appending text & aria-label.
+  const openLabelAttr = attachmentEl.getAttribute('data-cta-text');
+  const openLabel =
+    (openLabelAttr && openLabelAttr.trim()) ||
+    getLocalizationService(pageEl).getLocalizedString(
+      LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL
+    );
```

Comment:
Not sure what this will resolve to, maybe you're looking for `openLabelAttr ? openLabelAttr.trim() : getLocalizationService()...` as a ternary operator to understand what it's doing better